docs/users/games/Bannerlord.md

```diff
@@ -12,6 +12,7 @@ Get information on potential issues in your loadout(s). [Learn more about Health
 ![An example Health Check message prompting the installation of a missing mod.](../images/0.7.0/HealthCheckExampleBannerlord.webp)
 
 Diagnostics are shown in the following situations and uses the SubModule.xml file included with mods to generate these messages: 
+!!! warning "Bannerlord Software Extender (BLSE) is not supported in 0.7.1. This is planned for a future release."
```

Comment:
This can say 'next release', it's already been merged to main branch.

Reply:
Thanks but I'll just leave it as-is. That note can be removed entirely when I do the docs update for 0.7.2 :)